test/integration/test-video-players-helper.js

```diff
@@ -588,11 +589,12 @@ export function runVideoPlayerIntegrationTests(
     before(function() {
       this.timeout(TIMEOUT);
       // Skip autoplay tests if browser does not support autoplay.
-      return supportsAutoplay(window, false).then(supportsAutoplay => {
-        if (!supportsAutoplay) {
-          this.skip();
-        }
-      });
+      return VideoUtils.isAutoplaySupported(window, /* isLite */ false)
+        .then(supportsAutoplay => {
```

Comment:
Should this call the `skipIfAutoplayUnsupported ` helper function like the others? It doesn't look like it does anything different.

Reply:
It should! Thanks for catching.